CRM/Utils/ICalendar.php

```diff
@@ -51,6 +53,7 @@ public static function formatText($text) {
    * @return string
    */
   public static function unformatText($text) {
+    $text = str_replace("\n ", "", $text);
```

Comment:
Agree that universe suggests this is only used in tests.

In fact, only ICalendarTest, so moving this function into that file might make sense. But I'll wait until Justin rebases the other PR first since that will be funnier. (grin)